tools/progress.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: © 2023 Decompals
+# SPDX-License-Identifier: MIT
+
+from __future__ import annotations
+
+import argparse
+import mapfile_parser
```

Comment:
this script relies on mapfile_parser, which isn't listed on requirements.txt

Reply:
Thanks for the catch :)